components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -236,4 +304,10 @@ class GeckoEngineSession(
         internal const val MOZ_NULL_PRINCIPAL = "moz-nullprincipal:"
         internal const val ABOUT_BLANK = "about:blank"
     }
+
+    fun String.isPhone(): Boolean = contains("tel:", true)
```

Comment:
Nice, let's move these to our support-ktx module as standard string extensions? Seems like they could come in handy.

Reply:
I kept in here so that it wouldn't show up for all Strings outside this class. I don't think it makes sense for all strings to have an `isPhone`. I know extension functions don't add any overhead to the class since they're static methods really, but it seems misleading.

When we have a URI class (see #648) these exts should go there.

Reply:
Right, that's fair. I just wasn't sure if having a String ext function for single use makes sense...

Reply:
OTOH, our String extension also has an `isUrl` function. So, `isPhone` etc. would seem to fit there. The Strings wouldn't have a `isPhone` function unless `mozilla.components.support.ktx.kotlin.isPhone` is imported. So, it wouldn't show up for Strings by default. Anyways, don't feel strongly here, just a suggestion :).

Reply:
> The Strings wouldn't have a isPhone function unless `mozilla.components.support.ktx.kotlin.isPhone` is imported.

Oh, that's a good point! I'll move it there then.